pkg/upgraders/osd/upgrader.go

```diff
@@ -126,9 +126,14 @@ func PreClusterHealthCheck(c client.Client, cfg *osdUpgradeConfig, scaler scaler
 	if err != nil {
 		return false, err
 	}
+
 	desired := upgradeConfig.Spec.Desired
 	if upgradeCommenced {
-		logger.Info(fmt.Sprintf("ClusterVersion is already set to Channel %s Version %s, skipping %s", desired.Channel, desired.Version, upgradev1alpha1.UpgradePreHealthCheck))
+		if len(desired.Channel) > 0 && len(desired.Version) > 0 {
```

Comment:
This part of logging is moved to clusterversion package for now instead of dealing with it in the upgrader. That helps reduce some code. To be further improved as part of https://issues.redhat.com/browse/OSD-7686